tests/unit/files/files-manager.test.ts

```diff
@@ -0,0 +1,409 @@
+import { FilesManager } from '../../../src/files';
+import { FileResponse } from '../../../src/files/types';
+import { HttpClient } from '../../../src/http';
+import { MockHttpClient } from '../mocks';
+
+describe('FilesManager', () => {
```

Comment:
Just a silly comment, but since we have that for every other test, could we use the arrange/act/assert comments here too? :eyes: